install.sh

```diff
@@ -72,5 +72,5 @@ echo "# Emirge is now installed. Please run the following commands            #"
 echo "# to test the installation (assuming your shell is bash):               #"
 echo "# $ source $conda_prefix/bin/activate dgfem                             #"
 echo "# $ export OCL_ICD_VENDORS=$MY_CONDA_DIR/envs/dgfem/etc/OpenCL/vendors  #"
-echo "# $ python mirgecom/examples/wave-eager.py                              #"
+echo "# $ python /path/to/mirgecom/examples/wave-eager.py                     #"
 echo "#########################################################################"
```

Comment:
This set of instructions is a nice touch, but the box will typically come out misaligned:
```
#########################################################################
# Emirge is now installed. Please run the following commands            #
# to test the installation (assuming your shell is bash):               #
# $ source /home/andreask_work/miniforge3/bin/activate dgfem                             #
# $ export OCL_ICD_VENDORS=/home/andreask_work/miniforge3/envs/dgfem/etc/OpenCL/vendors  #
# $ python /path/to/mirgecom/examples/wave-eager.py                     #
#########################################################################
```
How about not having a boundary on the right?

Reply:
Fixed.